packages/app/src/embed/components/Content/index.tsx

```diff
@@ -363,6 +366,18 @@ export default class Content extends React.PureComponent<Props, State> {
     });
   };
 
+  refresh = () => {
+    if (this.preview && this.preview.handleRefresh) {
+      this.preview.handleRefresh();
+    }
+  };
+
+  openInNewWindow = () => {
+    // this is set in app/Preview
+    // i don't know why but I ain't complaining
+    if (window.openNewWindow) window.openNewWindow();
```

Comment:
@CompuIves Getting a typescript error here, but no clue how to fix it

```
src/embed/components/Content/index.tsx(376,38): error TS2339: Property 'openNewWindow' does not exist on type 'Window & typeof globalThis'.
```

Reply:
I agree. In this case a `@ts-ignore` is fine because this is a hack in the first place. If this wouldn't be a hack we could put in `global.ts` that `openNewWindow` is possibly on the `window`, and then we could even add some documentation to the field. But yeah, this is a real hacky way of circumventing prop passing 😅.